app/views/food_pickups/edit.html.erb

```diff
@@ -1,62 +1,66 @@
-  </div>
-
-  <div class="field">
-    <%= label_tag :location %><br />
-    <%= text_field_tag :location, current_donor.address %>
-  </div>
-
-  <div class="field">
-    <%= label_tag :phone_number %><br />
-    <%= text_field_tag  :phone_number, current_donor.phone_number %>
-  </div>
-
-  <div class="field">
-    <%= label_tag "Earliest Pick-up Time:" %><br />
-    <%= datetime_field_tag :start_time, @food_pickup.start_time.strftime('%Y-%m-%dT%I:%M:%S') %>
-  </div>
-
-  <div class="field">
-    <%= label_tag "Latest Pick-up Time:" %><br />
-    <%= datetime_field_tag :end_time, @food_pickup.end_time.strftime('%Y-%m-%dT%I:%M:%S') %>
-  </div>
-
-  <div class="field">
-    <%= label_tag "Reoccurs:" %><br />
-    <%= select_tag "reoccurrence", options_for_select(["None", "Daily", "Weekly", "Monthly", "Yearly"]) %>
-  </div>
-
-  <div class="field">
-    <%= label_tag "You will be charged #{number_to_currency(@food_pickup.donor.charge)}" %>
-  </div>
-  <% if current_donor.admin? %>
-    <div>
-      <%= label_tag :charge %><br />
-      <%= number_field_tag :charge, @food_pickup.donor.charge.to_i, :class => 'text_field', :step => 'any' %>
-    </div>
-    <div>
-      <%= label_tag "Approve Pickup:" %><br />
-      <%= select_tag :approved, options_for_select([["Approved", true], ["Unapproved", false]], selected: @food_pickup.approved) %>
-    </div>
-    <%= submit_tag "Approve" %>
-  <% else %>
-    <%= submit_tag "Update" %>
-  <% end %>
+<%= form_tag("/pickup/#{@food_pickup.id}/edit", method: "patch") do %>
+<div>
+  <img src="<%= @food_pickup.picture %>" alt="Food Picture" style="width=302px;height:228px">
+</div>
+
+<div class="field">
+  <%= label_tag :description %><br />
+  <%= text_field_tag :description, @food_pickup.description %>
+</div>
+
+<div class="field">
+  <%= label_tag :quantity %>(lbs)<br />
+  <%= number_field_tag :quantity, @food_pickup.quantity %>
+</div>
+
+<div class="field">
+  <%= label_tag :contact_name %><br />
+  <%= text_field_tag :contact_name, current_donor.contact_name %>
+</div>
+
+<div class="field">
+  <%= label_tag :location %><br />
+  <%= text_field_tag :location, current_donor.address %>
+</div>
+
+<div class="field">
+  <%= label_tag :phone_number %><br />
+  <%= text_field_tag  :phone_number, current_donor.phone_number %>
+</div>
+
+<div class="field">
+  <%= label_tag "Earliest Pick-up Time:" %><br />
+  <%= datetime_field_tag :start_time, @food_pickup.start_time.strftime('%Y-%m-%dT%I:%M:%S') %>
+</div>
+
+<div class="field">
+  <%= label_tag "Latest Pick-up Time:" %><br />
+  <%= datetime_field_tag :end_time, @food_pickup.end_time.strftime('%Y-%m-%dT%I:%M:%S') %>
+</div>
+
+<div class="field">
+  <%= label_tag "Reoccurs:" %><br />
+  <%= select_tag "reoccurrence", options_for_select(["None", "Daily", "Weekly", "Monthly", "Yearly"]) %>
+</div>
+
+<div class="field">
+  <%= label_tag "You will be charged #{number_to_currency(current_donor.charge)}" %>
+</div>
+
```

Comment:
We need the logic for if current donor is an admin around lines 50-59 because these are params that we only want the admin to change